.travis.yml

```diff
@@ -26,8 +26,12 @@ matrix:
 #  - if [ $TRAVIS_OS_NAME = osx ]; then brew install gcc; fi
 
 ## uncomment the following lines to override the default test script
+before_script:
+  - cd bin
+  - julia --project --color=yes -e 'import Pkg; Pkg.instantiate(); include("../.continuous-integration/check_generated_structs.jl")'
```

Comment:
It seems odd to include the script like this.  Can we put the project logic in the script?

Reply:
This will start the Julia session with the correct Manifest. I prefer this syntax since it separates the package manager aspects from the script itself.

Reply:
Can you remove the "cd bin" and then just run the command from the root of the repo?